### DISCUSSION TITLE
Search results have limited volume

### DISCUSSION THREAD

**Discussion by zhimin-z** [forem/forem#17723] [Q&A]
I attempt to type in the keyword onto the search bar. Let's say "machine learning". However, I can only find at most 60 results at most. This happens for any other keywords. Is there any way to access more posts or this is designed as such?
![image](https://user-images.githubusercontent.com/8592144/169200897-a5e9b5ea-794b-40f6-9ca7-75613da99fd8.png)

**Comment by JennieOcken** [comment]
I believe what you are looking at is the Chrome search function.  We don't control that or can do anything with it as it is controlled by the browser.  We are not doing anything to limit those search results.

**Reply by zhimin-z** [reply]
Chrome search function? I am using the internal search engine within DEV rather than anything else. Why do you say so?

**Reply by JennieOcken** [reply]
Are you looking at the search for `machine learning` or the search for `comments`?  Ah, I think I understand what you are saying.  You are using the Chrome search for "comment" to count how many results are on the page.  Very clever.  Let me look into this.  I expect it's intentional but I will find out for sure.

**Reply by zhimin-z** [reply]
Any update for the hidden results?
